qa/logging-config/custom-log4j2.properties

```diff
@@ -0,0 +1,187 @@
+#
```

Comment:
Can this properties file be named to make clear it is for v7?

Reply:
will do